DataStructures/HashTable/HashTable.cs

```diff
@@ -1,18 +1,23 @@
 ﻿using System;
+using System.ComponentModel;
 using System.Data;
 
 
 namespace DataStructures.HashTable
 {
     public class HashTable : IHashTable
     {
-        private object[] _array;
+        private class ArrayValue
```

Comment:
My suggestion is to rename ArrayValue to HashTableItem or something like that.

Reply:
Okay